server/src/domain/user/user.core.ts

```diff
@@ -98,7 +99,7 @@ export class UserCore {
     if (payload.storageLabel) {
       payload.storageLabel = sanitize(payload.storageLabel);
     }
-
+    payload.avatarColor = getRandomAvatarColor();
```

Comment:
I would prefer making this deterministic without having to explicitly save it in the database on user create.